week-1/exercises/initials.js

```diff
@@ -4,3 +4,5 @@ let lastName = "Johnson";
 
 // Declare a variable called initials that stores the first character of each string in upper case to form the user's initials
 // Log the variable in each case
+
+let initials = (firstName[0] + middleName[0] + lastName[0]).toUpperCase();
```

Comment:
Good job Fathi, but please could you explain the purpose of using the 'toUpperCase' method. And does it work with all initials or just with the initial of the last name?